### PR TITLE
Automatically serialize serde_json

### PR DESCRIPTION
**Pull request by mkawalec**
It's behind a feature enabled by default. Not bumping the version since we'll roll it into v5

### CODE REVIEW

**Review by sehz — APPROVED** [review]
Nice work!

**Comment by sehz** [comment]
rebase with master

**Comment by mkawalec** [comment]
rebased

File: nj-core/Cargo.toml [hunk 1]
```diff
@@ -11,6 +11,10 @@ license = "Apache-2.0"
 [lib]
 test = false
 
+[features]
+default = ["serde_json"]
```
Comment (nicholastmosher) [inline nj-core/Cargo.toml:15]: This seems like something that shouldn't be a default feature, though I suppose I don't have a super strong opinion on it.

Comment (mkawalec) [reply nj-core/Cargo.toml:15]: My personal use case was that serde_json values are very common when working with json in general and node/js in particular. But if you feel like serde is not something everyone is going to use often, I can make the feature optional.

Comment (sehz) [reply nj-core/Cargo.toml:15]: Let's make it optional for now.

Comment (mkawalec) [reply nj-core/Cargo.toml:15]: Sounds good

**Pull request merged**